pkg/cache/cluster.go

```diff
@@ -821,11 +817,7 @@ func (c *clusterCache) sync() error {
 
 	openAPISchema, gvkParser, err := c.kubectl.LoadOpenAPISchema(config)
 	if err != nil {
-		e, ok := err.(*kube.CreateGVKParserError)
-		if !ok {
-			return err
-		}
-		c.log.Info("warning: failed to create gvk parser", "error", e.Error())
+		return err
```

Comment:
Please add context in the returned error so we know which code path it came from.